lib/active_merchant/billing/gateways/cecabank/cecabank_json.rb

```diff
@@ -272,7 +273,7 @@ def encode_post_parameters(post)
       end
 
       def encode_params(params)
-        Base64.strict_encode64(params.to_json)
+        Base64.strict_encode64((params.is_a?(Hash) && params.to_json) || params)
```

Comment:
Question:
this `(params.is_a?(Hash) && params.to_json)` will return a boolean? 
If yes, that means you will encode a true or false value.